[2025-09-26T17:08:18.882+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-26T17:08:18.975+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.v2.0.hello_user scheduled__2025-09-16T03:00:00+00:00 [queued]>
[2025-09-26T17:08:19.008+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.v2.0.hello_user scheduled__2025-09-16T03:00:00+00:00 [queued]>
[2025-09-26T17:08:19.011+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-26T17:08:19.063+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): hello_user> on 2025-09-16 03:00:00+00:00
[2025-09-26T17:08:19.094+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag2.v2.0', 'hello_user', 'scheduled__2025-09-16T03:00:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp6r2rcf54']
[2025-09-26T17:08:19.112+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=216) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-26T17:08:19.115+0000] {standard_task_runner.py:72} INFO - Started process 237 to run task
[2025-09-26T17:08:19.114+0000] {standard_task_runner.py:105} INFO - Job 176: Subtask hello_user
[2025-09-26T17:08:19.393+0000] {task_command.py:467} INFO - Running <TaskInstance: dag2.v2.0.hello_user scheduled__2025-09-16T03:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-26T17:08:20.056+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2.v2.0' AIRFLOW_CTX_TASK_ID='hello_user' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-16T03:00:00+00:00'
[2025-09-26T17:08:20.069+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-26T17:08:20.202+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-26T17:08:20.238+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-26T17:08:20.241+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag2.v2.0, task_id=hello_user, run_id=scheduled__2025-09-16T03:00:00+00:00, execution_date=20250916T030000, start_date=20250926T170818, end_date=20250926T170820
[2025-09-26T17:08:20.338+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-26T17:08:20.446+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-26T17:08:20.452+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
